core/git_mixins/merge.py

```diff
@@ -35,11 +35,12 @@ def launch_tool_for_file(self, fpath):
         base_content, ours_content, theirs_content = versioned_content
 
         temp_dir = tempfile.mkdtemp()
+        repo_path = self.repo_path
```

Comment:
This change is to make sure `self.repo_path` property is only called once and we definitely use the same path below.